searchboost_warden/Dockerfile

```
@@ -1,25 +1,41 @@
 FROM rust:1.93.1-slim AS builder
 
+ARG BUILD_PROFILE=release
+
 RUN apt-get update && apt-get install -y \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)